### PR TITLE
remove events key from payload

### PR DESCRIPTION
**Pull request by andistuder**
this seems added as a duplication by the rails controller

and we do not use it

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -23,4 +23,10 @@ def unauthenticated_strategy
     self.status = 403
     self.response_body = 'Forbidden'
   end
+
+  def event_payload(parameters)
+    event_payload = parameters.dup
+    event_payload.delete 'event'
+    event_payload
```
Comment (dideler) [inline app/controllers/events_controller.rb]: Can reduce the code with [`Hash#except` from Rails](http://api.rubyonrails.org/classes/Hash.html#method-i-except)

**Comment by dideler** [comment]
Interesting, maybe some side effect from Rails STI?

File: app/controllers/events_controller.rb (hunk 1 above)
Comment (andistuder) [inline app/controllers/events_controller.rb]: thanks! good idea

**Comment by dideler** [comment]
LGTM

**Pull request merged**